src/content/changes/highlights/2021-07-16.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2021-07-16T10:00:00
+title: Highlights for the week ending July 16, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Managed apps
+
+[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Potentially Breaking**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable, view [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) for details.
```

Comment:
```suggestion
[NGINX IC v2.0.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/) is upgraded to controller container image v0.48.1. It contains performance improvements related to the admission webhook. **Potentially Breaking change**: For security, it disables forwarding traffic to ExternalNameServices by default. To enable it please check the details in the [changelog](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v2.0.0/).
```

@pipo02mix sorry I overrode your suggestion. This OK to you?

Added clarification in the main changelog too https://github.com/giantswarm/nginx-ingress-controller-app/pull/213

Reply:
yes cool!